packages/app/components/login.component.tsx

```diff
@@ -110,7 +113,7 @@ export const Login = () => {
           : `bankid:///?autostarttoken=${token}&redirect=null`
       Linking.openURL(bankIdUrl)
     } catch (err) {
-      setError('Öppna BankID manuellt')
+      setError(translate('bankid_OpenManually'))
```

Comment:
Should this be `'auth.bankid_OpenManually'`?

Reply:
NICE!